exhaustive.go

```diff
@@ -212,6 +228,7 @@ func init() {
 // Flag names used by the analyzer. They are exported for use by analyzer
 // driver programs.
 const (
+	ExplicitExhaustiveSwitchFlag   = "explicit-exhaustive-switch"
```

Comment:
What do you think of using a shorter name: `explicit`?

Reply:
That would work. Although it's maybe useful to clarify here between looking at switches as explicitly exhaustive vs e.g. looking at enums as explicitly exhaustive.

```
// only run exhaustive checks on defined types marked for enforcement
//exhaustive:enforce
type MyEnum int

//exhaustive:enforce
type MyExternalEnumAlias = someexternalpkg.Enum
```

It's not something currently supported by this analyzer, but maybe it will be one day.

Reply:
Thanks, this makes a lot of sense.